ext/gmp/gmp.c

```diff
@@ -586,6 +586,33 @@ ZEND_MODULE_INFO_D(gmp)
 }
 /* }}} */
 
+static zend_result convert_zstr_to_gmp(mpz_t gmp_number, const zend_string *val, zend_long base, uint32_t arg_pos)
```

Comment:
Side note: Normally I would avoid refactoring like this in a patch.

Reply:
I would too, but copy pasting this in the constructor seems rather pointless and I cannot really use the macro as is in the constructor instead

Reply:
Oh lol sorry I missed this is now used twice, that makes sense of course.